io/zenoh-transport/src/unicast/universal/rx.rs

```diff
@@ -197,7 +197,7 @@ impl TransportUnicastUniversal {
                 .decode()
                 .map_err(|_| zerror!("{}: decoding error", link))?;
 
-            log::trace!("Received: {:?}", msg);
+            // log::trace!("Received: {:?}", msg);
```

Comment:
This trace is useful for debugging.

Reply:
I tried to keep it away from the hotpath....